composer.json

```diff
@@ -14,19 +14,22 @@
         "php": ">=5.4",
         "psr/http-message": "^1.0",
         "php-http/message-factory": "^1.0.2",
-        "clue/stream-filter": "^1.3"
+        "clue/stream-filter": "^1.3",
+        "slim/slim": "^3.5"
```

Comment:
Not sure I understand you. Also, please see my previous comment about moving the removal to before_install instead of before_script with --no-update. --no-update causes to update composer.json, but does not actually call install.